{"name": "Тест авторизации с неверным логином и паролем", "status": "passed", "description": "Получение ошибки. Авторизация не выполнена", "start": 1718628658983, "stop": 1718628659260, "uuid": "46aeabf8-8bac-4d91-8729-f40a9ac42d64", "historyId": "7d8ee533535a2ed61f3922b2354a854f", "testCaseId": "7d8ee533535a2ed61f3922b2354a854f", "fullName": "tests.test_api_login_user.TestLoginUser#test_login_user_incorrect_email_and_password_error", "labels": [{"name": "feature", "value": "Проверка авторизации пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "pc"}, {"name": "thread", "value": "4920-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api_login_user"}]}